packages/vats/src/provisionPool.js

```diff
@@ -158,16 +220,32 @@ export const start = (zcf, privateArgs) => {
       });
   };
 
-  const creatorFacet = Far('Provisioning Pool creator', {
-    makeHandler: makeBridgeProvisionTool(sendInitialPayment),
+  // For documentation and/or future upgrade to Far
```

Comment:
pretty easy to use `makeFarHeapInstance` rn, up to you

Reply:
I tried once without luck. I'm interested to try again with a little help.

Reply:
approving now but happy to work with you to get the change in if this ends up needing another push

Reply:
https://github.com/Agoric/agoric-sdk/compare/6067_governance...ta/6067_governance